mod/metrics/set/abstract/metric_answer/research.rb

```diff
@@ -82,18 +81,23 @@ def answer_delete_button
     smart_link_to "Delete",
                   type: "button",
                   path: { action: :delete, success: success },
-                  class: "btn btn-outline-danger pull-right",
+                  class: "btn btn-sm btn-outline-danger pull-right",
                   'data-confirm': confirm,
                   "data-disable-with": "Deleting"
   end
 
   view :edit_buttons do
     class_up "form-group", "w-100 m-3"
     button_formgroup do
-      [standard_submit_button, standard_cancel_button, answer_delete_button]
+      [standard_save_button, cancel_research_button, answer_delete_button]
```

Comment:
cancel needed to be changed to reload page; otherwise the right side did not correct itself.